src/runtime/Module.cpp

```diff
@@ -368,17 +368,19 @@ Instance* Module::instantiate(ExecutionState& state, const ExternVector& imports
 #if !defined(NDEBUG)
 void ModuleFunction::dumpByteCode()
 {
-    printf("function type %p\n", m_functionType);
-    printf("requiredStackSize %u, requiredStackSizeDueToLocal %u\n", m_requiredStackSize, m_requiredStackSizeDueToLocal);
-
+    printf("\n");
+    printf("required stack size: %u bytes\n", m_requiredStackSize);
+    printf("required stack size due to local: %u bytes\n", m_requiredStackSizeDueToLocal);
+    printf("bytecode size: %zu bytes\n", m_byteCode.size());
+    printf("\n");
     size_t idx = 0;
     while (idx < m_byteCode.size()) {
         ByteCode* code = reinterpret_cast<ByteCode*>(&m_byteCode[idx]);
-        printf("%zu: ", idx);
-
+        printf("%6zu ", idx);
         switch (code->opcode()) {
 #define GENERATE_BYTECODE_CASE(name, ...)    \
     case ByteCode::name##Opcode:             \
+        printf(#name " ");                   \
```

Comment:
Is there any advantage of printing the name separately? Easier to maintain?

Reply:
Thus, there is no need to write this command at each bytecode class, which results shorter code.
Moreover, the name of the bytecode is linked to the name used by the macro. If something changes, at less places it's needed to change to name.

Or is it too much oversimplifying, and I should revert it?